playbook/app/pb_kits/playbook/pb_selectable_card/_selectable_card.tsx

```diff
@@ -131,43 +131,41 @@ const SelectableCard = (props: SelectableCardProps) => {
           className={globalProps(labelProps)}
           htmlFor={inputIdPresent}
       >
-        <div className="buffer">
```

Comment:
all i did was remove the buffer div but the line numbers changed making the diff look nuts